client/src/css/style.css

```diff
@@ -0,0 +1,324 @@
+@import url("./stylesheet.css");
+
+*{
```

Comment:
same thing about spaces before `{` in this file (we should decide what is normal for our code and use one way of formatting for whole project)